### DISCUSSION TITLE
Authenticator extension is preview on Safari

### DISCUSSION THREAD

**Discussion by Sneezry** [Authenticator-Extension/Authenticator#828] [General]
Please have a try and send us feedback! https://apps.apple.com/app/authen/id1602945200

![image](https://user-images.githubusercontent.com/1621293/148335740-062cd13c-318b-4d0b-b2c6-e4ceb3ff7a0a.png)

**Comment by rebornix** [comment]
I have just enabled backup support and released a new version 1.1 to the store. Along with the backup fix, the icon for the macOS app was also tweaked a bit

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/876920/151680797-d8940a84-b1ec-465d-bb9e-41f4da37c9f3.png">

**Comment by smitshj** [comment]
My phone has been stolen, my Google Authenticator was on it, I have several programs that I have to open with my Google Authenticator, so that doesn't work, how can I get into my programs?
